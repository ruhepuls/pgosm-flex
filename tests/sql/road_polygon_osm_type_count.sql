SELECT osm_type COLLATE "C", COUNT(*)
    FROM osm.road_polygon
    GROUP BY osm_type COLLATE "C"
    ORDER BY osm_type COLLATE "C"
;
