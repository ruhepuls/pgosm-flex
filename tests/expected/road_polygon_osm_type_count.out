bridleway|1
elevator|3
footway|29
living_street|1
pedestrian|73
service|1
services|1
steps|1
